src/utils.rs

```diff
@@ -167,17 +167,27 @@ pub fn check_case(input: &str, identifier: &str, case: Case) -> ScaffoldResult<(
             "{identifier} must be {case:?} Case",
         )));
     }
+    if input.chars().next().map_or(false, char::is_numeric) {
+        return Err(ScaffoldError::InvalidStringFormat(format!(
+            "{identifier} must not start with a numeric"
```

Comment:
```suggestion
            "{identifier} must not start with a number"
```

Or maybe "numeric value"?